packages/hint-validate-summary/src/hint.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @fileoverview check for summary tag and look for the computedStyle to check whether it is the same as it should be
+ */
+
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { IHint } from 'hint/dist/src/lib/types';
+import { StyleEvents, StyleParse } from '@hint/parser-css';
+import meta from './meta';
+const selectorParser = require('postcss-selector-parser');
```

Comment:
If we change this to an `import` then we get the types automatically (so we don't have to specify them ourselves later).
```suggestion
import selectorParser = require('postcss-selector-parser');
```